lib/ansible/modules/cloud/amazon/cloudformation_exports.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: cloudformation_exports
+short_description: Read a value from CloudFormation Exports
+description:
+  - Module retrieves a value from CloudFormation Exports
+requirements: ['boto3 >= 1.11.15']
+version_added: "2.10"
+author:
+  - "Michael Moyle (@mmoyle)"
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+- name: Get Exports
+  cloudformation_exports:
+    profile: 'my_aws_profile'
+    region: 'my_region'
+  register: cf_exports
+- debug: var=cf_exports
```

Comment:
```suggestion
- debug: 
    msg: "{{ cf_exports }}"
```
We generally prefer this syntax.